tests/unit/modules/rh_service_test.py

```diff
@@ -0,0 +1,329 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Jayesh Kariya <jayeshk@saltstack.com>`
+'''
+
+# Import Python Libs
+from __future__ import absolute_import
+
+# Import Salt Testing Libs
+from salttesting import TestCase, skipIf
+from salttesting.mock import (
+    MagicMock,
+    patch,
+    NO_MOCK,
+    NO_MOCK_REASON
+)
+
+from salttesting.helpers import ensure_in_syspath
+
+ensure_in_syspath('../../')
+
+# Import Salt Libs
+from salt.modules import rh_service
+
+# Globals
+rh_service.__salt__ = {}
+
+RET = ['hostname', 'mountall', 'network-interface', 'network-manager',
+       'salt-api', 'salt-master', 'salt-minion']
+
+FLAG = None
```

Comment:
@jayeshka, would you mind renaming this variable to something like HAS_UPSTART instead of FLAG?  Thanks.